mlflow/server/js/src/experiment-tracking/pages/experiment-evaluation-datasets/components/ExportTracesToDatasetModal.tsx

```diff
@@ -0,0 +1,216 @@
+import {
+  Checkbox,
+  Empty,
+  Input,
+  Modal,
+  SearchIcon,
+  Table,
+  TableCell,
+  TableHeader,
+  TableRow,
+  TableSkeletonRows,
+  useDesignSystemTheme,
+} from '@databricks/design-system';
+import { ColumnDef, flexRender, getCoreRowModel, useReactTable } from '@tanstack/react-table';
+import { FormattedMessage } from 'react-intl';
+import { useInfiniteScrollFetch } from '../hooks/useInfiniteScrollFetch';
+import { useSearchEvaluationDatasets } from '../hooks/useSearchEvaluationDatasets';
+import { EvaluationDataset } from '../types';
+import { getTrace } from '../../../utils/TraceUtils';
+import { useCallback, useEffect, useState } from 'react';
+import { getModelTraceId, ModelTrace } from '@mlflow/mlflow/src/shared/web-shared/model-trace-explorer';
+import { compact } from 'lodash';
+import { extractDatasetInfoFromTraces } from '../utils/datasetUtils';
+import { useUpsertDatasetRecordsMutation } from '../hooks/useUpsertDatasetRecordsMutation';
+import { CreateEvaluationDatasetButton } from './CreateEvaluationDatasetButton';
+
+const CheckboxCell: ColumnDef<EvaluationDataset, string>['cell'] = ({ row }) => {
```

Comment:
you might want to use `<TableRowSelectCell>` instead of custom component
https://ui-infra.dev.databricks.com/storybook/js/packages/du-bois/index.html?path=/docs/primitives-tableui--docs#selectable-rows